<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StabDynamicTestExecute</class>
 <widget class="QWidget" name="StabDynamicTestExecute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frMain">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>2</red>
            <green>2</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>2</red>
            <green>2</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>2</red>
            <green>2</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="lblProbeTitle">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Стабилометрический динамический тест</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frClient">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frSKG">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>4</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="SKGWidget" name="wgtSKG" native="true">
               <widget class="QLabel" name="lblFrontComment">
                <property name="geometry">
                 <rect>
                  <x>130</x>
                  <y>230</y>
                  <width>341</width>
                  <height>71</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Bookman Old Style</family>
                  <pointsize>22</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frControl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="btnZeroing">
               <property name="text">
                <string>Центровка</string>
               </property>
               <property name="icon">
                <iconset resource="../../kernel.qrc">
                 <normaloff>:/images/Zeroing.png</normaloff>:/images/Zeroing.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frScale">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="lblScale">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>2</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Масштаб</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbScale">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRecord">
               <property name="text">
                <string>Запись</string>
               </property>
               <property name="icon">
                <iconset resource="../../kernel.qrc">
                 <normaloff>:/images/Save.png</normaloff>:/images/Save.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblRecLenTitle">
               <property name="text">
                <string>Длительность записи</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblRecLen">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>00:00</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="pbRec">
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frSelectChannel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="lblSelectChannel">
                  <property name="text">
                   <string>Выбор канала</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbSelectChannel"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frValues">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="lblX">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>X = 0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblY">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Y = 0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblZ">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Z = 0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbShowTrace">
               <property name="text">
                <string>Рисовать след</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbAdvChannels">
               <property name="text">
                <string>Каналы</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frSpecific">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7"/>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnCalibrate">
               <property name="text">
                <string>Калибровка</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="AdvancedChannels" name="wgtAdvChannels" native="true"/>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCommunicationError">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ошибка связи с устройством:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedChannels</class>
   <extends>QWidget</extends>
   <header location="global">advancedchannels.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SKGWidget</class>
   <extends>QWidget</extends>
   <header location="global">skgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbAdvChannels</sender>
   <signal>clicked(bool)</signal>
   <receiver>StabDynamicTestExecute</receiver>
   <slot>on_advChannelsClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>1058</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnZeroing</sender>
   <signal>clicked()</signal>
   <receiver>StabDynamicTestExecute</receiver>
   <slot>zeroing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>1054</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRecord</sender>
   <signal>clicked()</signal>
   <receiver>StabDynamicTestExecute</receiver>
   <slot>recording()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>1059</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StabDynamicTestExecute</receiver>
   <slot>scaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>985</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>1056</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_advChannelsClicked(bool)</slot>
  <slot>zeroing()</slot>
  <slot>recording()</slot>
  <slot>scaleChange(int)</slot>
 </slots>
</ui>
